### DISCUSSION TITLE
Installazione HACS - wget bad address

### DISCUSSION THREAD

**Discussion by lucaob** [indomus/forum#1431] [GENERALE - Discussioni generiche]
Ciao,
mi sto accingendo ad installare HA sul mio raspberry nella modalità HA Core su Docker ma ho problemi nell'installazione di HACS.
Per farlo sto seguendo questa guida: https://indomus.it/guide/installare-hacs-home-assistant-community-store-sul-proprio-hub/
Mi trovo nella sezione "HOME ASSISTANT CORE (SU DOCKER, SU RASPBERRY PI OS)".
Fin'ora credo di aver seguito tutto senza problemi ma quando digito il seguente comando:
`wget -O - https://get.hacs.xyz | bash -`
ottengo questo errore:
`wget: bad addres 'get.hacs.xyz'`
Purtroppo non sono assolutamente esperto quindi non so a cosa sia dovuto o come risolvere il problema.
Potete aiutarmi?

Grazie,
Luca

**Comment by snakuzzo** [comment]
`wget -O - https://get.hacs.xyz | bash -`

**Reply by lucaob** [reply]
Sì, ho scritto -O, non -0 eppure se digito il comando restituisce sempre lo stesso errore. Se, però, copio ed incollo quello che mi hai scritto tu allora funziona. A me sembrano identici.
Dunque ho risolto il problema ma non so perché!

Grazie, comunque.
Luca

**Reply by snakuzzo** [reply]
secondo me il "trattino" con il copia incolla dalla guida ti ha dato problemi

**Reply by lucaob** [reply]
E' possibile ma io ho ridigitato il comando a manina ed il problema lo dava lo stesso. Invece copiando ed incollando dal tuo messaggio qui sopra ha funzionato al primo colpo. Vai a capire...
